src/main/com/wsscode/pathom3/connect/planner.cljc

```diff
@@ -920,6 +925,16 @@
           shape))
       true)))
 
+(>defn shape-statically-unreachable?
```

Comment:
not sure if we need this function, seems a bit just a reverse of `shape-reachable?`